bottery/platform/__init__.py

```diff
@@ -41,8 +41,9 @@ def webhook_endpoint(self):
     def webhook_url(self):
         return 'https://{}{}'.format(settings.HOSTNAME, self.webhook_endpoint)
 
+    @property
+    def tasks(self):
+        return getattr(self, '__tasks__', [])
```

Comment:
Creating `__attr__` attributes is not a good idea, if you wanted to create a private attr, `_attr` would be a better name. But for this case, I believe a class attribute would be good enough.